pyproject.toml

```diff
@@ -15,107 +34,20 @@ extend-exclude = '''
-
-# DEVELOPER INSTALL INSTRUCTIONS (assuming current directory is at the top of the project) :
-# Poetry:
-#   poetry install --extras dev
-# PIP:
-#   pip install -e .[dev]
-
-[tool.poetry]
-name = "screenpy_selenium"
-version = "4.0.4"
-description = "ScreenPy extension to enable interacting with Selenium."
-authors = ["Perry Goy <perry.goy@gmail.com>"]
-maintainers = ["Gabe Langton", "Marcel Wilson"]
-license = "MIT"
-repository = "https://github.com/ScreenPyHQ/screenpy_selenium"
-documentation = "https://screenpy-selenium-docs.readthedocs.io"
-readme = "README.md"
-classifiers = [
-    "Operating System :: OS Independent",
-    "Programming Language :: Python :: 3 :: Only",
-    "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9",
-    "Programming Language :: Python :: 3.10",
-    "Programming Language :: Python :: 3.11",
-    "Programming Language :: Python :: 3.12",
-    "Development Status :: 5 - Production/Stable",
-    "Intended Audience :: Developers",
-    "Topic :: Software Development :: Quality Assurance",
-    "Topic :: Software Development :: Testing",
-    "Topic :: Software Development :: Testing :: BDD",
-    "Natural Language :: English",
-    "License :: OSI Approved :: MIT License",
-]
-
-# It's possible to add optional dependencies with the poetry CLI tool using:
-#   poetry add --optional some_dev_pkg
-# This will set `optional = true` flag in [tool.poetry.dependencies], as seen below
-# But then you need to remember to manually edit the [tool.poetry.extras] dev section
-# and declare the package. This allows `pip install .[dev]` to work as expected
-# Poetry users will need to use the `--extras dev` option rather than the `--with dev`
-# so we dont have two different sets of package versions to update.
-
-[tool.poetry.dependencies]
-python = "^3.8"
-screenpy = ">=4.0.2"
-screenpy_pyotp = ">=4.0.0"
-selenium = ">=4.1.0"
-importlib_metadata = {version = "*", python = "3.8.*"}
-
-# convenience packages for development of screenpy only
-black = {version = "*", optional = true}
-coverage = {version = "*", optional = true}
-mypy = {version = "*", optional = true}
-pre-commit = {version = "*", optional = true}
-pytest = {version = "*", optional = true}
-ruff = {version = ">=0.2.0", optional = true}
-sphinx = {version = "*", optional = true}
-tox = {version = "*", optional = true}
-
-[tool.poetry.extras]
-dev = [
-    "pre-commit",
-    "pytest",
-    "tox",
-]
-dev_all = [
-    "black",
-    "coverage",
-    "mypy",
-    "pre-commit",
-    "pytest",
-    "ruff",
-    "sphinx",
-    "tox",
-]
-test = [
-    "coverage",
-    "pytest",
-]
-
-[build-system]
-requires = ["poetry-core>=1.2.0"]
-build-backend = "poetry.core.masonry.api"
-
 
 [tool.ruff]
 target-version = "py38"  # minimum supported version
 line-length = 88  # same as Black.
-
 extend-exclude = [
-    "screenpy_selenium/__init__.py",
-    "screenpy_selenium/__version__.py",
+    "__init__.py",
```

Comment:
I probably don't need to comment on this on every one. I missed the one on ScreenPy: PyOTP after all. But same thing here—we probably want to remove this exclusion.

Reply:
Haha. Maybe that's why i had those files ignored.

We could instead just ignore the specific rules in those files in the `per-file-ignore` section. I don't want to completely turn off `ruff`ing them up, and especially not for *all* of the `__init__.py` files.

Reply:
Updated!  I was also able to remove the blanket exclusion for `__version__.py` in each of the projects.